src/main.ts

```diff
@@ -69,62 +73,113 @@ function fillWorld() { // temp
 	}
 }
 
+const widgetsTex = new Texture("tiny/textures/gui/widgets.png");
+
 function draw() {
 	game.width  = innerWidth
 	game.height = innerHeight
 	game.style.width  = innerWidth  + "px"
 	game.style.height = innerHeight + "px"
 
-	const ctx = game.getContext("2d")
+	const ctx = game.getContext("2d") as CanvasRenderingContext2D;
```

Comment:
```suggestion
	const ctx: CanvasRenderingContext2D = game.getContext("2d")
```
Not important tho, it's already in main branch